docs/user-guide/building-assistants.rst

```diff
@@ -185,7 +185,7 @@ To test our model against the test file, run the command:
 
 .. code-block:: bash
 
-   rasa test --e2e --stories test_stories.md
+   rasa test --e2e --stories tests/test_stories.md
```

Comment:
```suggestion
   rasa test --stories tests/test_stories.md
```
happens by default now. in fact i think that it might look in the `tests` folder by default, removing the need for the second arg? (sorry for commenting on not your code)